src/api/app/models/workflow/step/rebuild_package.rb

```diff
@@ -0,0 +1,39 @@
+class Workflow::Step::RebuildPackage < ::Workflow::Step
+  include Triggerable
+
+  REQUIRED_KEYS = [:project, :package].freeze
+
+  attr_reader :project_name, :package_name
+
+  def call(options = {})
+    return unless valid?
+
+    workflow_filters = options.fetch(:workflow_filters, {})
+    # Call Triggerable method to set all the elements needed for rebuilding
+    set_project_name
+    set_package_name
+    set_project
+    set_package
+    set_object_to_authorize
+    set_multibuild_flavor
+
+    Pundit.authorize(@token.user, @token, :rebuild?)
+    # Perform rebuild
+    rebuild_package(workflow_filters)
+  end
+
+  def set_project_name
+    @project_name = step_instructions[:project]
+  end
+
+  def set_package_name
+    @package_name = step_instructions[:package]
+  end
+
+  private
+
+  def rebuild_package(_workflow_filters = {})
```

Comment:
We don't need workflow filters, so let's remove it.

```suggestion
  def rebuild_package
```